src/skin.cpp

```diff
@@ -245,16 +245,11 @@ void SkinInfillAreaComputation::applySkinExpansion(const Polygons& original_outl
  *
  * this function may only read/write the skin and infill from the *current* layer.
  */
-void SkinInfillAreaComputation::generateSkinInsets(SliceLayerPart* part)
+void SkinInfillAreaComputation::generateSkinInsetsAndInnerSkinInfill(SliceLayerPart* part)
```

Comment:
Well maybe it's just the wording.

The general working of the polygon processor is as follows:
- we get an area which needs to be filled in some way
- we split that area up into two parts which need to be filled differently from each other
- recurse until finished ^^

This is exactly what happens here: we split up the outline of the skin area and produce both the insets and the infill.